.claude/agents/software-engineer.md

```
@@ -7,14 +7,7 @@ model: sonnet
 
 ## Project-specific local supplement
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)